compilation errors in simd code

```

In file included from ../src/lib/sw_engine/tvgSwRaster.cpp:220:
../src/lib/sw_engine/tvgSwRasterAvx.h: In function 'bool avxRasterTranslucentRect(SwSurface*, const SwBBox&, uint8_t, uint8_t, uint8_t, uint8_t)':
../src/lib/sw_engine/tvgSwRasterAvx.h:92:35: error: request for member 'join' in 'surface->SwSurface::blender', which is of non-class type 'SwBlender' {aka 'uns
igned int (*)(unsigned int, unsigned int, unsigned char)'}
   92 |     auto color = surface->blender.join(r, g, b, a);
      |                                   ^~~~
../src/lib/sw_engine/tvgSwRasterAvx.h: In function 'bool avxRasterTranslucentRle(SwSurface*, const SwRleData*, uint8_t, uint8_t, uint8_t, uint8_t)':
../src/lib/sw_engine/tvgSwRasterAvx.h:141:35: error: request for member 'join' in 'surface->SwSurface::blender', which is of non-class type 'SwBlender' {aka 'un
signed int (*)(unsigned int, unsigned int, unsigned char)'}
  141 |     auto color = surface->blender.join(r, g, b, a);
      |                                   ^~~~
../src/lib/sw_engine/tvgSwRasterAvx.h:151:23: error: 'IALPHA' was not declared in this scope
  151 |         auto ialpha = IALPHA(src);
      |                       ^~~~~~
```

--- a/src/lib/sw_engine/tvgSwRasterAvx.h
+++ b/src/lib/sw_engine/tvgSwRasterAvx.h
@@ -89,7 +89,7 @@ static bool avxRasterTranslucentRect(SwSurface* surface, const SwBBox& region, u
         return false;
     }
 
-    auto color = surface->blender.join(r, g, b, a);
+    auto color = surface->join(r, g, b, a);
     auto buffer = surface->buf32 + (region.min.y * surface->stride) + region.min.x;
     auto h = static_cast<uint32_t>(region.max.y - region.min.y);
     auto w = static_cast<uint32_t>(region.max.x - region.min.x);
@@ -138,7 +138,7 @@ static bool avxRasterTranslucentRle(SwSurface* surface, const SwRleData* rle, ui
         return false;
     }
 
-    auto color = surface->blender.join(r, g, b, a);
+    auto color = surface->join(r, g, b, a);
     auto span = rle->spans;
     uint32_t src;
 
@@ -148,7 +148,7 @@ static bool avxRasterTranslucentRle(SwSurface* surface, const SwRleData* rle, ui
         if (span->coverage < 255) src = ALPHA_BLEND(color, span->coverage);
         else src = color;
 
-	auto ialpha = IALPHA(src);
+	auto ialpha = IA(src);
 
         //1. fill the not aligned memory (for 128-bit registers a 16-bytes alignment is required)
         auto notAligned = ((uintptr_t)dst & 0xf) / 4;

